internals/secrethub/generate.go

```diff
@@ -74,7 +85,56 @@ func (cmd *GenerateSecretCommand) before() error {
 		return err
 	}
 
-	cmd.generator = randchar.NewGenerator(useSymbols)
+	charset := randchar.Alphanumeric
+	if useSymbols {
+		symbols, found := randchar.CharsetByName("symbols")
+		if found {
+			charset = charset.Add(symbols)
+		} else {
+			return ErrCouldNotFindCharSet("symbols")
+		}
+	}
+
+	var includedCharsets []randchar.Charset
+	for _, charsetName := range cmd.includes {
+		charsetToInclude, found := randchar.CharsetByName(charsetName)
+		if !found {
+			return ErrCouldNotFindCharSet(charsetName)
+		}
+		charset = charset.Add(charsetToInclude)
+		includedCharsets = append(includedCharsets, charsetToInclude)
+	}
+
+	for _, charsetName := range cmd.excludes {
+		charsetToExclude, found := randchar.CharsetByName(charsetName)
+		if !found {
+			return ErrCouldNotFindCharSet(charsetName)
+		}
+		charset = charset.Subtract(charsetToExclude)
+		for i, includedCharset := range includedCharsets {
+			if charsetToExclude.Equals(includedCharset) {
+				return ErrFlagsMutuallyExclusive(cmd.includes[i], charsetName)
+			}
+		}
+	}
```

Comment:
We discussed moving the include/exclude mutually exclusive check to the `randchar` package, but ended up questioning whether we should have include/exclude flags or just override the charset entirely when you do not want to use the default charset.

@Marton6 is writing a proposal for this in #177